cmd/minishift/cmd/start_test.go

```diff
@@ -361,11 +362,11 @@ func TestDetermineIsoUrl(t *testing.T) {
 		in  string
 		out string
 	}{
-		{"", constants.DefaultB2dIsoUrl},
-		{"b2d", constants.DefaultB2dIsoUrl},
-		{"B2D", constants.DefaultB2dIsoUrl},
-		{"centos", constants.DefaultCentOsIsoUrl},
-		{"CentOs", constants.DefaultCentOsIsoUrl},
+		{"", minishiftUtil.GetLatestISOVersion("b2d")},
```

Comment:
I meant the change as a whole and in particular just retrieving the latest version based on the latest tag on the release page. 

We need more than this in order to keep control over compatibility. As mentioned in the issue, the idea is to be using semantic versioning.  

What we want to achieve is to introduce some sort of independency between Minishift release and ISO release. Aka, if we build Minishift against 1.1.1 and then we release we screed something up in the ISO and we release 1.1.2, Minishift should transparently download this version, without having to release a new Minishift version.

On the other hand, if we release a ISO 2.0.0 with some backward incompatible changes (potentially even requiring changes in the Minishift side), this ISO should not be downloaded automatically.